### ISSUE TITLE
Crash when scanning barcode to start consuming from startsreen. 

### ISSUE THREAD

**Issue by VNRARA** [patzly/grocy-android#705]
Crash when consuming from homescreen. 

```
--------- beginning of crash
05-22 15:11:37.889 23237 23237 E AndroidRuntime: FATAL EXCEPTION: main
05-22 15:11:37.889 23237 23237 E AndroidRuntime: Process: xyz.zedler.patrick.grocy, PID: 23237
05-22 15:11:37.889 23237 23237 E AndroidRuntime: java.lang.ClassCastException
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at androidx.camera.core.impl.Config$-CC.m(SourceFile:34)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.helper.DownloadHelper.$r8$lambda$zJNAjc_nZe4lkBmnJTJMiR-c_e8(SourceFile:33)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at androidx.camera.view.PreviewView$1$$ExternalSyntheticLambda0.onResponse(SourceFile:109)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.helper.DownloadHelper$$ExternalSyntheticLambda7.onResponse(Unknown Source:9)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.web.CustomStringRequest$$ExternalSyntheticLambda0.onResponse(SourceFile:27)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.web.CustomStringRequest.deliverResponse(SourceFile:11)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at com.google.android.gms.common.images.zaa.run(SourceFile:210)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:942)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8757)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
05-22 15:11:37.889 23237 23237 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
--------- beginning of main
05-22 15:11:42.975 24018 24018 E r.patrick.grocy: Not starting debugger since process cannot load the jdwp agent.
```

**Comment by dominiczedler** [comment]
Should be fixed with the latest version 3.2.1.
Please test it with this version (I don't know which version you currently use).

**Comment by VNRARA** [comment]
Just updated. So if you don't hear from me on here its good.
